src/main/java/ee/tuleva/onboarding/payment/provider/PaymentInternalReferenceService.java

```diff
@@ -11,7 +11,7 @@
 
 @Service
 @RequiredArgsConstructor
-class PaymentInternalReferenceService {
+public class PaymentInternalReferenceService {
```

Comment:
Does the visibility need to be made public? Or can it be package-private? (i.e. this was intentionally encapsulated to just 1 package, not accidental).

Reply:
Yup, unfortunately we had to make lots of services public, as Montonio package is another package.